src/bosh-director/lib/clouds/external_cpi_response_wrapper.rb

```
@@ -36,37 +36,18 @@ def invoke_cpi_method(method, *arguments)
-      final_args = args.take(2)
-      if @cpi_api_version >= 3
-        final_args = args
-      end
-      return  @cpi.create_stemcell(*final_args)
+    def create_stemcell(*args)
+      final_args = @cpi_api_version >= 3 ? args : args.take(2)
+      @cpi.create_stemcell(*final_args)
     end
 
-
     def attach_disk(*args)
```

RuboCop (1.88.0)

[warning] Method `Bosh::Clouds::ExternalCpiResponseWrapper#attach_disk` is defined at both src/bosh-director/lib/clouds/external_cpi_response_wrapper.rb:23 and src/bosh-director/lib/clouds/external_cpi_response_wrapper.rb:47.

(Lint/DuplicateMethods)